README.md

```diff
@@ -22,7 +22,7 @@ authenticates Terraform, letting you get started with less setup.
 
 1. Clone this repository:
 
-   git clone https://github.com/GoogleCloudPlatform/terraform-docs-samples.git
+   git clone [https://github.com/terraform-google-modules/terraform-docs-samples.git](https://github.com/terraform-google-modules/terraform-docs-samples.git)
```

Comment:
Thank you for noticing this! I'm going to correct this further, as this is a command line prompt, it shouldn't be a link: 
```suggestion
    `git clone https://github.com/terraform-google-modules/terraform-docs-samples`
```

Reply:
Heartfeltly acknowledged @glasnt - for the review and remediation of my [untidy act](https://github.com/terraform-google-modules/terraform-docs-samples/pull/776), which from the bounteous and redeeming nature of the *Computational Grace* from the `GCP`, was fortunately merged into the otherwise pristine and spellbinding [codebase](https://github.com/terraform-google-modules/terraform-docs-samples).

Well, not that it should be a reprise or an extension of the initial *Carta de Amor*, how do you find the palatability of my proposal, to further extend Pull Request #776 in updating the [README.md](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/README.md), to correct a very minor typographical omission in the [Providing feedback](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/README.md#providing-feedback) section, where the existing ~~Alternativly~~ should be beautified to Alternatively.

In my celestial stupor of *Rastafarian Levitation*, the propensity for haste and finality inadvertently eclipsed my *Wandering Eye*, from noticing additional cosmetic features that required beautification through the [same act](https://github.com/terraform-google-modules/terraform-docs-samples/pull/776).

¿If at all, this cosmetic smudge can be beautified internally through your *finer touch* under this very [same act](https://github.com/terraform-google-modules/terraform-docs-samples/pull/776), or @arnoldmashava will have to pull up another stunt again and deliver much tidier *Carta de Amor*?

Podría ser un acto delicado, pero por ahora, adjunto mi agradecimiento y Amor:

```dart
>_ sudo ¡Muchas Gracias! --verbosity=debug
```

Rastafari.

Reply:
```
>>> import glasnt
>>> glasnt.reply(verbosity=DEFAULT)
```
If you find other spelling errors in `README.md`, please submit another PR with them all batched together, and I'll approve/merge it. 

```
>>> glasnt.thanks()
```
Thank you for noticing these issues, and for submitting changes to improve the repo!

Reply:
Indeed @glasnt - well, as for the succinctness and brevity:

```python
>>> glasnt.reply(verbosity=DEFAULT)
```

I will rollback my ambitious update to the `stable` channel:

```python
>>> rastafari.upload("shy_smile")
```

Gracias y Gracia...